modules/microshift-config-parameters-table.adoc

```diff
@@ -77,25 +77,46 @@ The following table explains {microshift-short} configuration YAML parameters an
 |`number`
 |By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
 
+|`ingress.certificate.Secret`
+|`string`
```

Comment:
better to remove the last "." from "ingress.certificate.Secret", so it should be `ingress.certificateSecret`.